content/includes/stories.yaml

```diff
@@ -0,0 +1,161 @@
+      icon_height: 119
+      class: fast
+    - title@: Open
+      description@: AMP stories are part of the open web and can be shared and embedded across sites and apps without being confined to a single ecosystem.
+      icon: /static/img/stories/icon-open.png
+      icon_width: 87
+      icon_height: 119
+      class: open
+features:
+  title@: AMP story features
+  features:
+    - description@: Supports rich visuals with native video and image performance
+      image: /static/img/stories/features-wired.gif
+      image_width: 600
+      image_height: 581
+      image_alt@: A Pod Races Through the Hyperloop for the First Time
+    - description@: Enriches storytelling through animations and tappable interactions
+      image: /static/img/stories/features-people.gif
+      image_width: 600
+      image_height: 581
+      image_alt@: A Pod Races Through the Hyperloop for the First Time
+    - description@: Works across mobile and desktop devices on the open web
+      image: /static/img/stories/features-hearst.gif
+      image_width: 600
+      image_height: 581
+      image_alt@: A Pod Races Through the Hyperloop for the First Time
+    - description@: Builds on top of AMP’s component library to support features like analytics
+      image: /static/img/stories/features-amp.png
+      image_width: 600
+      image_height: 581
+      image_alt@: A Pod Races Through the Hyperloop for the First Time
+news:
+  cards:
+    - url: /content/learn/case-studies/washingtonpost.md
+      quote@: “As a source for breaking news and information, this format allows us to showcase our quality journalism when there are multiple elements we want to bring together. Combining reporting, photography, videos and motion graphics, this gives readers a more visual entry point when they are searching for our coverage.”
+      source@: Greg Manifold, Design Director
+      image: /static/img/stories/wapo-logo.png
+      image_width: 284
+      image_height: 44
+      class: wapo
+    - url: /content/learn/case-studies/washingtonpost.md
+      quote@: “AMP stories have incredible potential to enhance how we create mobile content. Our team can easily create beautiful, media-rich stories that our users will now be able to access quickly across the web. We continue to be focused on creating the best content for every platform and the creative possibilities on AMP stories are endless”
+      source@: Doug Parker, Vice President of Digital Design
+      image: /static/img/stories/time-logo.png
+      image_width: 116
+      image_height: 35
+      class: time
+web_stories:
+  title@: Check out AMP stories on the web
+  stories:
+    - base_image: /static/img/stories/story-cnn
+      image_width: 385
+      image_height: 641
+      immage_alt@: The Missions Still Exploring Our Solar System
+      base_logo: /static/img/stories/logo-cnn
+      logo_width: 395
+      logo_height: 45
+      logo_alt@: CNN
+      url: https://stamps.cnn.io/solar-system-missions/index.html
```

Comment:
This URL should not be a staging URL.  Need to change.

Reply:
Change URL to https://www.cnn.com/ampstories/us/still-missing-cassini-heres-what-else-is-out-there